rotkehlchen/chain/evm/tokens.py

```diff
@@ -353,8 +353,13 @@ def _detect_erc721_tokens(
                     ))
                     filtered_tokens.append(token)
 
-            valid_tokens, results = [], self.evm_inquirer.multicall(calls=calls)
-            for token, result in zip(filtered_tokens, results, strict=False):
+            valid_tokens, outputs = [], self.evm_inquirer.multicall_2(calls=calls, require_success=False)  # noqa: E501
+            for token, output in zip(filtered_tokens, outputs, strict=False):
+                status, result = output
```

Comment:
does this work? I know there is a way but I always mess with it

```suggestion
            for token,(status, result) in zip(filtered_tokens, outputs, strict=False):
```